data/account/participation.go

```diff
@@ -203,19 +202,24 @@ func (part PersistedParticipation) Persist() error {
 	voting := part.Voting.Snapshot()
 	rawVoting := protocol.Encode(&voting)
 
-	return part.Store.Atomic(func(ctx context.Context, tx *sql.Tx) error {
+	err := part.Store.Atomic(func(ctx context.Context, tx *sql.Tx) error {
 		err := partInstallDatabase(tx)
 		if err != nil {
-			return fmt.Errorf("Participation.persist: failed to install database: %v", err)
+			return fmt.Errorf("failed to install database: %w", err)
 		}
 
 		_, err = tx.Exec("INSERT INTO ParticipationAccount (parent, vrf, voting, firstValid, lastValid, keyDilution) VALUES (?, ?, ?, ?, ?, ?)",
 			part.Parent[:], rawVRF, rawVoting, part.FirstValid, part.LastValid, part.KeyDilution)
 		if err != nil {
-			return fmt.Errorf("Participation.persist: failed to insert account: %v", err)
+			return fmt.Errorf("failed to insert account: %w", err)
 		}
 		return nil
 	})
+
```

Comment:
The change in this method ensures that in all the error cases, we will have the "PersistedParticipation.Persist" text as part of the error ( i.e. including if the error was generated by the Commit method )